examples/research_projects/codeparrot/scripts/arguments.py

```diff
@@ -148,6 +148,16 @@ class PreprocessingArguments:
     alpha_frac: Optional[float] = field(
         default=0.25, metadata={"help": "Maximum fraction of non-alphanumeric characters, otherwise file is filtered."}
     )
+    min_token_ratio: Optional[float] = field(
+        default=1.5, metadata={"help": "Minimum character token ratio for the file, otherwise file is filtered."}
+    )
+    filter_proba: Optional[float] = field(
+        default=0.7, metadata={"help": "Probability for filtering config, test and uncommon files."}
+    )
+    tokenizer_dir: Optional[str] = field(
```

Comment:
Since this does not strictly need to be a directory:

```suggestion
    tokenizer_name_or_dir: Optional[str] = field(
```
or alternatively:
```suggestion
    tokenizer: Optional[str] = field(
```